README.md

```diff
@@ -1,201 +1,51 @@
-# News challenge
+## Welcome to the Daily Jordan News!
 
-* Feel free to use Google, your notes, books, etc. but work on your own.
-* If you refer to the solution of another coach or student, please put a link to that in
-  your README.
-* If you have a partial solution, **still check in a partial solution** and send in a pull
-  request.
-* You must submit a pull request to this repo with your code by 10am Monday morning.
+Using my web app you will be able to view the hottest daily headlines. If you would like to search for a specific news article, or news subject, you can do so in the search bar.
 
-## Challenge
+## How to use
```

Comment:
You could add instructions on how users can add their own api key to test the page